accounts/README.md

```diff
@@ -1,45 +1,73 @@
 # Create Accounts!
 
+A couple of things to help us along:
 
-Ngā mihi nui kia koutou katoa! Warm greetings to you all! Welcome to Dev Academy.
-
-You will need to set up these accounts before starting Phase 0. The general rule of thumb is to **use your first name as part of your username wherever possible** and include a photo of yourself (i.e. not your cat or bike). This helps us (the EDA community of staff, teachers, coaches and alumni) to better support you.
+1. **Use your first name as part of your username wherever possible.** (To help us all recognise each other)
+2. **Open links in new tabs.** GitHub repos have a lot of links. We recommend right-clicking (or holding the command key (mac) / control key (win) when clicking) to open links in new tabs.
+3. **Skim read TLDR sections.** We use TLDR to indicate the information is useful but not essential for completing the task right now.
```

Comment:
TLDR stands for "Too long don't read", which is intended to be a summary of a much longer read. I don't think we're using this initialism the correct way in these documents. Perhaps we should just title these sections as "Context"?

Reply:
Context is good. Thanks @don-smith